EnlightenMAUI.csproj

```diff
@@ -188,6 +191,12 @@
 	</ItemGroup>
 
 	<ItemGroup>
+	  <None Update="CHANGELOG.md">
+	    <CopyToOutputDirectory>Never</CopyToOutputDirectory>
+	  </None>
+	  <None Update="dist\background_model.tflite">
+	    <CopyToOutputDirectory>Never</CopyToOutputDirectory>
+	  </None>
```

Comment:
Just making sure this was deliberate.

Reply:
It was not but will be removed in the next PR I'll be posting shortly